apps/showcase/src/assets/trainings/sdk/steps/typescript-sdk/exercise/app.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, inject, signal } from '@angular/core';
+import { PetApi } from '../../../../libs/sdk/src/api';
+import type { Pet } from '../../../../libs/sdk/src/models';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent {
+  /** Title of the application */
+  public title = 'tutorial-app';
+
+  private readonly petStoreApi = inject(PetApi);
+  public pets = signal<Pet[]>([]);
```

Comment:
```suggestion
  readonly #pets = signal<Pet[]>([]);
  public readonly pets = this.#pets.asReadonly();
```

(mainly to expose `Signal` and not `writableSignal`, not for the fancy `#` private prefix :))